### PR TITLE
Support for Foundry 0.8.x

### PR DESCRIPTION
**Pull request by The-E**
Merged in the other two PRs (#13 and #12) that were unmerged on the origin repo and corrected a small issue found in testing where some variables were cast to string unnecessarily.

### CODE REVIEW

**Comment by Ourobor** [comment]
I'll have a look on my test stuff when I get a chance. Thanks!

File: module.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "author": "Ourobor",
   "version": "0.5.5",
   "minimumCoreVersion": "0.6.0",
```
Comment (BoltsJ) [inline module.json:7]: This should be bumped to 0.8.6 so that it doesn't install for people still on 0.7.10 or earlier.

**Comment by Ourobor** [comment]
Sorry I haven't had time to work on this until now. There are additional issues with the previous two pull requests that need resolution(though it's possible your change to remove the cast to strings fixes one of them). 

Interestingly enough, it wasn't unnecessary, it wouldn't work at all in previous versions of foundry without casting them to strings... for some reason :| I have absolutely no clue why, but it wanted strings so I gave it strings.  

Anyway, my current plan to move forward is to create a new branch (086Support) and merge all currently open PRs into that branch so we can have all progress in one place and so I can also add fixes I've noticed as well without being some weird mess of confusing pull requests. I'm also working on making a tag for 0.7.x stuff so the legacy code is still available for people running old foundry when 086Support eventually merges into master/main

**Comment by Ourobor** [comment]
All changes have been merged into 086Support. If you have more changes, please open a new PR against that branch. Given that it's a development branch, it should be a lot easier to merge in changes since the burden isn't on you to make everything perfect.

Thank you for your assistance so far!

**Pull request closed without merging**